trusona-sdk-resources/src/main/java/com/trusona/sdk/resources/dto/TrusonaficationResult.java

```diff
@@ -19,36 +19,24 @@
   private final String boundUserIdentifier;
   private final Date createdAt;
   private final Date updatedAt;
-
-  @Deprecated
```

Comment:
why did we remove this deprecated constructor ?

Reply:
Because it was deprecated /shrug